internal/snmp/config.go

```diff
@@ -9,7 +9,8 @@ type ClientConfig struct {
 	Timeout config.Duration `toml:"timeout"`
 	Retries int             `toml:"retries"`
 	// Values: 1, 2, 3
-	Version uint8 `toml:"version"`
+	Version                 uint8 `toml:"version"`
+	UseUnconnectedUDPSocket bool  `toml:"use_unconnected_udp_socket"`
```

Comment:
What do you think about leaving off the "use"? It makes the name longer but doesn't add anything. The other settings are nouns, so it doesn't fit to have this one be a verb phrase. The "use" came from gosnmp which doesn't have the same settings naming style.

```suggestion
	UnconnectedUDPSocket bool  `toml:"unconnected_udp_socket"`
```

Reply:
I like the idea. I was just following goSNMP. I will implement it